ironfish/src/memPool/recentlyEvictedCache.ts

```diff
@@ -0,0 +1,225 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { BufferMap } from 'buffer-map'
+import { Assert } from '../assert'
+import { createRootLogger, Logger } from '../logger'
+import { MetricsMonitor } from '../metrics'
+import { getTransactionSize } from '../network/utils/serializers'
+import { Transaction, TransactionHash } from '../primitives/transaction'
+import { PriorityQueue } from './priorityQueue'
+
+interface EvictionEntry {
+  hash: TransactionHash
+  feeRate: bigint
+}
+
+interface ExpirationEntry {
+  hash: TransactionHash
+  expiration: number
+}
+
+export class RecentlyEvictedCache {
+  private readonly metrics: MetricsMonitor
+  private readonly logger?: Logger
+
+  private readonly capacity: number = 1000 // TODO: fix this default initialization
+
+  private readonly transactions = new BufferMap<Transaction>()
+  /**
+   * Keep track of number of bytes stored in the transaction map
+   * */
+  private transactionsBytes = 0
```

Comment:
I think we might be able to get rid of this. We keep track of this in mempool because transaction entries can be variable size but here they are all the same size right? So this could be deduced from the total size

Reply:
removed